api/src/Entity/Camp.php

```diff
@@ -72,6 +72,7 @@ class Camp extends BaseEntity implements BelongsToCampInterface, CopyFromPrototy
      */
     #[Assert\Valid]
     #[Assert\Count(min: 1, groups: ['create'])]
+    #[Assert\Count(min: 2, minMessage: 'A camp must have at least one period.', groups: ['Period:delete'])]
```

Comment:
I think it's a perfectly sensible validation. To delete a period, there must be at least two periods present. Maybe it would be clearer to read if you changed the validation message to something like "Cannot delete the last period of a camp."
Although, that could also be misinterpreted as "I can't delete the chronologically last period in a multi-period camp"